frontend/src/routes/Routes.tsx

```diff
@@ -35,14 +35,17 @@ export enum PATHS {
   VIEW_EVENT_ID = '/schedule/events/view/:eventId',
   // FACILITY BOOKING
   FACILITY_BOOKING_MAIN = '/facility',
-  VIEW_FACILITY = '/facility/view',
+  VIEW_FACILITY = '/facility/view/',
   VIEW_FACILITY_ID = '/facility/view/:facilityID',
-  CREATE_FACILITY_BOOKING = '/facility/booking/create',
-  CREATE_FACILITY_BOOKING_ID = '/facility/booking/create/:facilityId',
+  CREATE_FACILITY_BOOKING = '/facility/booking/create/',
+  CREATE_FACILITY_BOOKING_DAILY_VIEW = '/facility/booking/dayview/create/',
+  CREATE_FACILITY_BOOKING_DAILY_VIEW_ID = '/facility/booking/dayview/create/:facilityID',
+  CREATE_FACILITY_BOOKING_ID = '/facility/booking/create/:facilityID',
   EDIT_FACILITY_BOOKING = '/facility/booking/edit/:bookingId',
   VIEW_FACILITY_CONFLICT = '/facility/booking/create/conflict/:facilityID',
   VIEW_FACILITY_BOOKING = '/facility/booking/view/:bookingId',
-  VIEW_FACILITY_BOOKING_DAILY_VIEW = '/facility/booking/dayview',
+  VIEW_FACILITY_BOOKING_DAILY_VIEW = '/facility/booking/dayview/',
+  VIEW_FACILITY_BOOKING_DAILY_VIEW_ID = '/facility/booking/dayview/:facilityID',
   VIEW_FACILITY_BOOKING_ID = '/facility/booking/view/',
```

Comment:
do help update this as well! might want to trace the usages and update those too